src/components/EmptyStateComponent/index.tsx

```diff
@@ -93,37 +95,41 @@ function EmptyStateComponent({
             addBottomSafeAreaPadding={addBottomSafeAreaPadding}
             addOfflineIndicatorBottomSafeAreaPadding={addOfflineIndicatorBottomSafeAreaPadding}
         >
-            <View style={[styles.skeletonBackground, styles.overflowHidden]}>
-                <SkeletonComponent
-                    gradientOpacityEnabled
-                    shouldAnimate={false}
-                />
-            </View>
+            {!!SkeletonComponent && (
+                <View style={[styles.skeletonBackground, styles.overflowHidden]}>
+                    <SkeletonComponent
+                        gradientOpacityEnabled
+                        shouldAnimate={false}
+                    />
+                </View>
+            )}
             <View style={styles.emptyStateForeground}>
-                <View style={styles.emptyStateContent}>
+                <View style={[styles.emptyStateContent, cardStyles]}>
                     <View style={[styles.emptyStateHeader(headerMediaType === CONST.EMPTY_STATE_MEDIA.ILLUSTRATION), headerStyles]}>{HeaderComponent}</View>
-                    <View style={shouldUseNarrowLayout ? styles.p5 : styles.p8}>
+                    <View style={[shouldUseNarrowLayout ? styles.p5 : styles.p8, cardContentStyles]}>
                         <Text style={[styles.textAlignCenter, styles.textHeadlineH1, styles.mb2, titleStyles]}>{title}</Text>
                         <Text style={[styles.textAlignCenter, styles.textSupporting, styles.textNormal]}>{subtitle}</Text>
                         {children}
-                        <View style={[styles.gap2, styles.mt5, !shouldUseNarrowLayout ? styles.flexRow : undefined]}>
-                            {buttons?.map(({buttonText, buttonAction, success, icon, isDisabled}, index) => (
-                                <View
-                                    // eslint-disable-next-line react/no-array-index-key
-                                    key={index}
-                                    style={styles.flex1}
-                                >
-                                    <Button
-                                        success={success}
-                                        onPress={buttonAction}
-                                        text={buttonText}
-                                        icon={icon}
-                                        large
-                                        isDisabled={isDisabled}
-                                    />
-                                </View>
-                            ))}
-                        </View>
+                        {!!buttons && (
```

Comment:
This expression will return `true` for empty array so you have to check its length
```suggestion
                        {buttons.length > 0 && (
```